flow/layers/child_scene_layer.cc

```diff
@@ -21,14 +21,18 @@ void ChildSceneLayer::Paint(PaintContext& context) const {
 void ChildSceneLayer::UpdateScene(SceneUpdateContext& context) {
   FML_DCHECK(needs_system_composite());
 
-  // TODO(MZ-191): Set clip.
+  // TODO(SCN-191): Set clip.
   // It's worth asking whether all children should be clipped implicitly
   // or whether we should leave this up to the Flutter application to decide.
   // In some situations, it might be useful to allow children to draw
   // outside of their layout bounds.
+  FML_LOG(ERROR) << "ChildSceneLayer::UpdateScene";
```

Comment:
DCHECK that at most one of the two is non-null?

Or, just say:

if (view_holder_proxy_) instead of "else if".  The former is probably better, if that's the expectation.

Reply:
Done